### ISSUE TITLE
finally block not executed without catch block

### ISSUE THREAD

**Issue by weihsiu** [rssh/dotty-cps-async#20]
unless catch block is present, finally block is not executed.
```
def runFuture(): Unit =
    val r = async[Future]:
      try
        val s = await(Future("hello"))
        val i = await(Future(throw new Exception))
        s * i
      catch
        case _: Exception => "caught"
      finally
        println("done")
    println(Await.result(r, Duration.Inf))
```
if the catch block is commented out, "done" is not printed.

**Comment by rssh** [comment]
thanks.

**Comment by rssh** [comment]
Fixed in 0.2.1-RC1 which is published as:
   "com.github.rssh" %% "dotty-cps-async" % "0.2.1-RC1"
